polyfill/lib/calendar.mjs

```diff
@@ -158,6 +223,22 @@ class ISO8601Calendar extends Calendar {
     ({ year, month, day } = ES.RegulateDate(year, month, day, overflow));
     return new constructor(year, month, day, this);
   }
+  timeFromFields(fields, options, constructor) {
```

Comment:
I also didn't think of this before, but should the ISO calendar have an implementation of `timeAdd`, `timeSubtract`, and `timeUntil`?